src/dex/definitions/sundaeswap-v3/order.ts

```diff
@@ -0,0 +1,96 @@
+import { DatumParameterKey } from '@app/constants';
+
+export default {
+  constructor: 0,
+  fields: [
+    {
+      constructor: 0,
+      fields: [
+        {
+          bytes: DatumParameterKey.PoolIdentifier,
+        },
+      ],
+    },
+    {
+      constructor: 0,
+      fields: [
+        {
+          bytes: DatumParameterKey.SenderPubKeyHash,
```

Comment:
constructor: 0,
      

This should be stake key hash. fields: [
        {
          bytes: DatumParameterKey.SenderStakingKeyHash,
        },
      ],